pages/filterCards/index.tsx

```diff
@@ -0,0 +1,78 @@
+import axios from "axios";
+// tslint:disable no-submodule-imports
+import { withRouter } from "next/router";
+import * as React from "react";
+
+interface Props {
+  data: {
+    tags: any;
+  };
+}
+
+interface State {
+  selectedTags: any;
+  tags: any;
+}
+
+class Index extends React.Component<Props, State> {
+  public state = {
+    selectedTags: {} as any,
+    tags: {} as any
+  };
+
+  public async componentDidMount() {
+    const response = await axios.get(`/api/cards/tags`);
```

Comment:
try / catch?

Reply:
@vvscode ,
could you, please, advise how to use it correctly? Show an error, that data doesn't come and try later? What do you think?
```
 public async componentDidMount() {
    try {
      const response = await axios.get(`/api/cards/tags`);
      this.setState({ tags: response.data.tags });
    } catch (e) {
      // what to place here ?
    }
  }
```

Reply:
show the error looks fine for me